sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -345,4 +345,50 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_accelerometer</EFDB_Topic>
+    <Description>LabJack connected accelerometer unit.</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The name of the sensor.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time at which the data was determined (TAI unix seconds).</Description>
```

Comment:
Consider:
Starting time of data acquisition (TAI unix seconds)

Reply:
Also, don't we need to measurement interval to be published as well?

Reply:
Added interval, please see changes.

Reply:
Timestamp description is generic, let's start a PR to change that for all.